docs/source/estimation.rst

```diff
@@ -35,31 +35,28 @@ argument of any criterion function optimized with estimagic's :ref:`minimize`.
 
 It can have the following columns (most of them being optional)
 
-- 'value' (dtype=float): Start values for the optimization. If not provided,
-  start values based on constraints and other information in params are drawn
-  randomly. If provided, we ensure that these values    are taken as start
-  values for local optimizers and are in the initial population of genetic
-  algorithms or pseudo global optimizers.
-- 'lower' (dtype=float): Lower bounds enforced in optimization. Can take the
+- `'value'`` (dtype=float): Start values for the optimization. This column is
```

Comment:
Here are a lot of unbalanced code ticks.

Reply:
Thanks, fixed.